### PR TITLE
프리뷰 ui  제작

### PR DESCRIPTION
**Pull request by GiPyoo**
## Related Issue

>  #8 

## Description and Motivation
스크린샷 2019-12-09 오후 3 28 48
위와 같습니다.

## Types of changes

- [ ] Docs change
- [ ] dependency upgrade
- [ ] refactoring
- [ ] Bug fix 
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Review

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: client/src/presentation/pages/snug/message-section/content.tsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ export const MessageSectionContent: React.FC<AppSocketChannelMatchProps> = props
     <MessageContextProvider>
       <MessageSectionContentWrapper>
         <ChatContent {...props} />
+        <Preview></Preview>
```
Comment (raccoonback) [inline client/src/presentation/pages/snug/message-section/content.tsx]: 해당 채널에 가입된 유저(프로필)인 경우, Preview 안보이게 하는 로직은 프로필 기능이 완료되면 추가되는 것이지요?

File: client/src/presentation/components/snug/chat-container/index.tsx [hunk 2]
```diff
@@ -9,13 +9,14 @@ import { usePathParameter } from "contexts/path-parameter-context";
 const ChatContentWrapper = styled.section.attrs({
   id: "scroll"
 })`
-  min-height: 90%;
-  max-height: 90%;
+  min-height: ${props => css`calc(100% - 75px)`};
+  max-height: calc(100% - 75px);
```
Comment (sangwon21) [inline client/src/presentation/components/snug/chat-container/index.tsx]: calc 인상적입니다.

File: client/src/presentation/pages/snug/message-section/content.tsx (hunk 1 above)
Comment (sangwon21) [reply client/src/presentation/pages/snug/message-section/content.tsx]: 꼼꼼한 리뷰네요

Comment (GiPyoo) [reply client/src/presentation/pages/snug/message-section/content.tsx]: 넵

**Pull request merged**